OracleTuxedo/salt/samples/bankapp/README.md

```diff
@@ -0,0 +1,54 @@
+
+# Oracle Tuxedo `bankapp` Sample Container Image
+
+This sample extends the Oracle Tuxedo image by creating a "bankapp" application.
+
+## Prerequisites
+
+This image uses the Orace Tuxedo container image `oracle/tuxedo:latest` as its base. Please follow the [Oracle Tuxedo image](https://github.com/oracle/docker-images/tree/main/OracleTuxedo/core) documentation to build the container image before performing the operations described in this document.
+
+## How to Build the Image
```

Comment:
```suggestion
## How to build the image
```

Reply:
I had it originally as the below but our documentation team had it changed to the caps for the first letter "Build" and "Image". I reverted it back to the below:
## How to build the image